G/GibbsSeaWater/build_tarballs.jl

```diff
@@ -3,20 +3,24 @@
 using BinaryBuilder
 
 name = "GibbsSeaWater"
-version = v"3.5.0"
+version = v"3.5.1"
 
 # Collection of sources required to build GibbsSeaWater
 # note 3.05.0-4 is too old to be cross-compiled on FreeBSD
 sources = [
-    "https://github.com/TEOS-10/GSW-C/archive/d392e91cb63341f543ed1609c4eff613055ab3cb.zip" =>
-    "454c46ec00468aeba048fa4f3cee095f927925ab699b1d1b3a25a838cef2d22c",
+    ArchiveSource(
+        "https://github.com/TEOS-10/GSW-C/archive/d392e91cb63341f543ed1609c4eff613055ab3cb.zip", "454c46ec00468aeba048fa4f3cee095f927925ab699b1d1b3a25a838cef2d22c"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/GSW-C-*
-cc $CFLAGS -fPIC -c -O3 -Wall gsw_oceanographic_toolbox.c gsw_saar.c
-cc $LDFLAGS -fPIC -shared -o libgswteos.$dlext gsw_oceanographic_toolbox.o gsw_saar.o -lm
+CC=cc
+if [[ "${target}" == aarch64-apple-* ]]; then
+    CC="gcc"
+fi
+$CC $CFLAGS -fPIC -c -O3 -Wall gsw_oceanographic_toolbox.c gsw_saar.c
+$CC $LDFLAGS -fPIC -shared -o libgswteos.$dlext gsw_oceanographic_toolbox.o gsw_saar.o -lm 
```

Comment:
Again, you don't need this:
```suggestion
cc -fPIC -c -O3 -Wall gsw_oceanographic_toolbox.c gsw_saar.c
cc -fPIC -shared -o libgswteos.$dlext gsw_oceanographic_toolbox.o gsw_saar.o -lm 
```

What is needed to use clang as compiler for aarch64-darwin is https://github.com/JuliaPackaging/Yggdrasil/blob/c8d4f76148c1c82667bb8f6c3b442aed57d184a8/L/LLVMOpenMP/build_tarballs.jl#L63-L65

Reply:
Thank you for your advice @giordano.
I fixed it.